README.md

```diff
@@ -1,16 +1,15 @@
 ![StreetComplete](http://www.westnordost.de/streetcomplete/featureGraphic.png)
 
-StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data.
+StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data. The app is aimed at users who do not know anything about OSM tagging schemes but still want to
+contribute to the OpenStreetMap by surveying their neighbourhood (or other places as well).
 
-The found issues are presented to the user as markers on a map (like i.e. in Osmose) that each are
-solvable by filling out a simple form to complete/correct the information on site. The user's
-answer is then processed and directly uploaded into the OSM database in meaningful changesets in the name
-of the user's OSM account.
+ StreetComplete on its own locates places where survey will be helpful. The found places are shown as markers on a map. Each can be solved by walking to the location and filling in the app a simple form to complete/correct the information. For example a road without name will result in a marker with an appropriate quest. After tapping it "What is the name of this road?" question is shown, with a text field to answer it. More examples are shown in screenshots below.
+
+The user's answer is automatically processed and directly uploaded into the OSM database. Edits are done in meaningful changesets using user's OSM account.
 Since the app is meant to be used on a survey, it can be used offline and is
 economic with data usage.
 
-The app is aimed at users who do not know anything about OSM tagging schemes but still want to
-contribute to the OpenStreetMap by surveying their neighbourhood (or other places as well). Because of the target group, only those quests are shown which are answerable very clearly by asking one simple question and only those quests are created which contain very few false positives.
+To make app easy to use quests are limited to answerable by asking simple questions. Also, only quests with very few false positives are allowed. For example street name quest is handling also roads without real names - both by skipping roads with `noname=yes` tag set and by allowing user to specify that road is really unnamed.
```

Comment:
```suggestion
To make the app easy to use, quests are limited to those answerable by asking simple questions. 
```

> Also, only quests with very few false positives are allowed. For example street name quest is handling also roads without real names - both by skipping roads with `noname=yes` tag set and by allowing user to specify that road is really unnamed.

Would need a more thorough explanation what is meant by that. The second sentence is confusing. Explaining it properly would be too much for the README.